habitat_baselines/rl/models/simple_cnn.py

```diff
@@ -0,0 +1,138 @@
+import numpy as np
+import torch
+import torch.nn as nn
+
+from habitat_baselines.rl.ppo.utils import Flatten
+
+
+class SimpleCNN(nn.Module):
+    def __init__(self, observation_space, output_size):
+        super().__init__()
+        if "rgb" in observation_space.spaces:
+            self._n_input_rgb = observation_space.spaces["rgb"].shape[2]
+        else:
+            self._n_input_rgb = 0
+
+        if "depth" in observation_space.spaces:
+            self._n_input_depth = observation_space.spaces["depth"].shape[2]
+        else:
+            self._n_input_depth = 0
+
+        # kernel size for different CNN layers
+        self._cnn_layers_kernel_size = [(8, 8), (4, 4), (3, 3)]
+
+        # strides for different CNN layers
+        self._cnn_layers_stride = [(4, 4), (2, 2), (1, 1)]
+
+        if self._n_input_rgb > 0:
+            cnn_dims = np.array(
+                observation_space.spaces["rgb"].shape[:2], dtype=np.float32
+            )
+        elif self._n_input_depth > 0:
+            cnn_dims = np.array(
+                observation_space.spaces["depth"].shape[:2], dtype=np.float32
+            )
+
+        if self.is_blind:
+            self.cnn = nn.Sequential()
+        else:
+            for kernel_size, stride in zip(
+                self._cnn_layers_kernel_size, self._cnn_layers_stride
+            ):
+                cnn_dims = self._conv_output_dim(
+                    dimension=cnn_dims,
+                    padding=np.array([0, 0], dtype=np.float32),
+                    dilation=np.array([1, 1], dtype=np.float32),
+                    kernel_size=np.array(kernel_size, dtype=np.float32),
+                    stride=np.array(stride, dtype=np.float32),
+                )
+
+            self.cnn = nn.Sequential(
+                nn.Conv2d(
+                    in_channels=self._n_input_rgb + self._n_input_depth,
+                    out_channels=32,
+                    kernel_size=self._cnn_layers_kernel_size[0],
+                    stride=self._cnn_layers_stride[0],
+                ),
+                nn.ReLU(True),
+                nn.Conv2d(
+                    in_channels=32,
+                    out_channels=64,
+                    kernel_size=self._cnn_layers_kernel_size[1],
+                    stride=self._cnn_layers_stride[1],
+                ),
+                nn.ReLU(True),
+                nn.Conv2d(
+                    in_channels=64,
+                    out_channels=32,
+                    kernel_size=self._cnn_layers_kernel_size[2],
+                    stride=self._cnn_layers_stride[2],
+                ),
+                #  nn.ReLU(True),
```

Comment:
@abhiskk -- When refactoring the baseline code, I notice we are missing this ReLU.  There is non-linearity between the last conv and the FC layer!